sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/QueryBuilders/QueryBuilderSamples.cs

```diff
@@ -0,0 +1,163 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.DigitalTwins.Core.QueryBuilder;
+
+namespace Azure.DigitalTwins.Core.Samples
+{
+    /// <summary>
+    /// Samples for QueryBuilder.
+    /// </summary>
+    public static class QueryBuilderSamples
+    {
+        /// <summary>
+        /// Main method.
+        /// </summary>
+        public static void Main()
+        {
+            #region Snippet:DigitalTwinsQueryBuilderNonGeneric
+            new DigitalTwinsQueryBuilder();
+            new DigitalTwinsQueryBuilder<BasicDigitalTwin>();
+
+            // SELECT * FROM DigitalTwins
+            new DigitalTwinsQueryBuilder().GetQueryText();
+            new DigitalTwinsQueryBuilder<BasicDigitalTwin>().GetQueryText();
+            #endregion
+            #region Snippet:DigitalTwinsQueryBuilder
+
+            // SELECT * FROM DigitalTwins
```

Comment:
I would just use the default (BasicDigitalTwins) version to show you don't have to have a type declared to get `SELECT * FROM DigitalTwins`

Reply:
So basically, what I am trying to say is, only use the declared type if you are planning to use its properties in your method calls.